### ISSUE TITLE
v0.4/v0.3 error on base64/blobUrl

### ISSUE THREAD

**Issue by gimox** [futurepress/epub.js#740]
1) get base64 from server
2) convert to blobUrl
3) try bloburl, with  window.open(blobUrl) => it save a correct file.epub and work with a reader
4) try bloburl with epubjs 0.2x ok it work perfect
5) ok now i can say that bloburl has correct data.
5) try with 0.4. Error:  `renderTo` is undefined

renderTo is undefined book can not be opened
```
const book = ePub(blob);
this.rendition = book.renderTo('area') // error renderTo is not a function.
```

i inspect book and it's a promise not resolved, try to resolve it. and i can see more function inside, like opened, opening etc etc but not renderTo. No error but RenderTo is undefined i can't render the book.
```
const book = ePub(blob);
 book.then((result) => {
        console.log('ok', result);
      }, (error) => {
        console.log(error)
      });
```

try same code pointing to a 'epub', it work perfect, result has renderTo and i can render the book.
```
const book = ePub('test.epub');
 book.then((result) => {
        console.log('ok', result);
      }, (error) => {
        console.log(error)

      });
```

Ok  any idea to render blob ?

**Comment by jlag34** [comment]
@gimox  I believe I had a similar issue and solved it [here](https://github.com/futurepress/epub.js/issues/746)

**Comment by gimox** [comment]
thanks, it found that it work if i had options object as empty:
`this.book = new ePub(this.getBlob(), {})`

Unfortunately it not work on base64!!

**Comment by fchasen** [comment]
This should be fixed in epubjs@0.3.61
